src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -803,6 +804,7 @@ private static String processPageSourceForGodMode(String content) {
                 .replace(TestProperties.inst().TEST_UNREG_ACCOUNT, "${test.unreg}")
                 .replace(Config.SUPPORT_EMAIL, "${support.email}")
                 // today's date
+                .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "{*}")
```

Comment:
If this has something to do with sanitization, on account of #4244, let's put this on hold first.
On another note, @bamboo3250 it'd be great if you can do it ASAP.